Data/Sequence/Internal.hs

```diff
@@ -4300,7 +4348,11 @@ zipWith f s1 s2 = zipWith' f s1' s2'
     s2' = take minLen s2
 
 -- | A version of zipWith that assumes the sequences have the same length.
+#if __GLASGOW_HASKELL__ >= 800
+zipWith' :: HasCallStack => (a -> b -> c) -> Seq a -> Seq b -> Seq c
+#else
```

Comment:
No no no. This is an internal function and is only used when we know it will succeed.